labels.rb

```diff
@@ -66,8 +67,8 @@
   if options[:delete_labels]
     util.delete_repo_labels(repo_name, extra_labels) unless extra_labels.empty?
   end
-  if options[:fix_labels]
-    util.update_repo_labels(repo_name, incorrect_labels) unless incorrect_labels.empty?
-    util.add_repo_labels(repo_name, missing_labels) unless missing_labels.empty?
-  end
+  next unless options[:fix_labels]
```

Comment:
In this block next unless could be replaced with an elsif followin the initial delete labels option and execute the utils functions only if the fix_labels option has been specified